cfme/common/provider.py

```diff
@@ -240,9 +240,13 @@ def delete(self, cancel=True):
             invokes_alert=True)
         sel.handle_alert(cancel=cancel)
         if not cancel:
-            flash.assert_message_match(
-                'Delete initiated for 1 {} Provider from the CFME Database'.format(
```

Comment:
So, related to this, this is going to start cropping up in lots of places, appliance now has a cached ```product_name``` attribute. Feel free to use this. Most objects should have an appliance attribute now anyway as part of navmazing, so this should be come
```python
            flash.assert_message_match(
                'Delete initiated for 1 {} Provider from the {} Database'.format(
                    self.string_name, self.appliance.product_name))
```